packages/fs-bridge/src/utils.ts

```
@@ -1,89 +1,229 @@
 import pathe from "pathe";
 import { BridgePathTraversal } from "./errors";
 
-const DANGEROUS_CONTROL_CHARACTER_REGEX = /[\0\n\r]/;
+const MAX_DECODING_ITERATIONS = 10;
+const WINDOWS_DRIVE_LETTER_REGEX = /^[A-Z]:/i;
+const WINDOWS_DRIVE_REGEX = new RegExp(`${WINDOWS_DRIVE_LETTER_REGEX.source}[/\\\\]`, "i");
+// eslint-disable-next-line no-control-regex
+const CONTROL_CHARACTER_REGEX = /[\u0000-\u001F\u007F-\u009F]/u;
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)